src/components/TraderCard/TraderCardBody/TraderCardBody.js

```diff
@@ -367,7 +370,7 @@ const TraderCard = (props) => {
         {showSummary && (
           <UserSummary
             isCopyTrading={copyTrader || profitSharingProvider}
```

Comment:
Then you can remove isCopyTrading and quote and use `provider` in UserSummary instead.

Reply:
Done